install/kubernetes/tetragon/templates/_container_oci_hooks_setup.tpl

```diff
@@ -11,6 +11,7 @@
     - --local-install-dir={{  include "container.tetragonOCIHookSetup.installPath" . }}
     - --host-install-dir={{ .Values.tetragon.ociHookSetup.installDir }}
     - --oci-hooks.local-dir={{ include "container.tetragonOCIHookSetup.hooksPath" . }}
+    - --oci-hooks.fail-allow-namespaces={{ if .Values.tetragon.ociHookSetup.failAllowNamespaces }}{{ printf "%s,%s" .Release.Namespace .Values.tetragon.ociHookSetup.failAllowNamespaces }}{{ else }}{{ .Release.Namespace }}{{ end }}
```

Comment:
Actually, one more comment: Can we always add Tetragon's namespace here? If it's not there, it is certainly an issue.

Reply:
That's already the case in the current implementation:

If the user uses sets `failAllowNamespaces` to `my-namespace,your-namespace` then the line will read as `    - --oci-hooks.fail-allow-namespaces=my-namespace,your-namespace,tetragon` (assuming Tetragon is deployed in a namespaces called `tetragon`).

Otherwise if `failAllowNamespaces` is empty (as is the default), then the line will read as `    - --oci-hooks.fail-allow-namespaces=tetragon` (assuming Tetragon is deployed in a namespaces called `tetragon`).

So Tetragon's namespace is always added.

Reply:
Ah, right. Missed that part. Awesome, thanks!